src/main/resources/i18n/messages.properties

```diff
@@ -119,10 +119,10 @@ FoldTab.64=<html><strong>Default:</strong> Enabled if filename prettifying is en
 FoldTab.65=Monitor played status of files
 FoldTab.66=<html><strong>Default:</strong> Disabled<br><strong>Notes:</strong> Folders must have "Monitor played status of files" enabled to have their contents included in the New Media folder.</html>
 FoldTab.67=Do nothing
-FoldTab.68=Add an overlay to the thumbnail
-FoldTab.69=Hide the file
-FoldTab.70=Move the file to a different folder
-FoldTab.71=Move the file to the recycle/trash bin
+FoldTab.68=Tag media
```

Comment:
@SubJunk I don't mean to "force my will" in any way. To me "media" made more sense, but that might just be for me. Maybe mediafile is even better?

Regarding option 1, I don't know which renderers shows thumbnails when you browse but from those I have used with UMS yet, only the PS3 does. In that list is my Onkyo Receiver, Foobar2000, MediaMonkey, WMP, VLC, various Android DLNA applications, a Panasonic 50" smart TV (don't remember model) and a Bluray player (Sony I think). My collection of renderers might not be representative, but I don't think it's correct to disregard it as only a fallback. It's not that I they the two are equally fancy, I just think it should be obvious to the users what the option do.